bin/ck8s

```diff
@@ -28,6 +28,7 @@ usage() {
     echo "  ops helmfile <wc|sc>                        run helmfile as cluster admin" 1>&2
     echo "  s3cmd [cmd]                                 run s3cmd" 1>&2
     echo "  user-kubeconfig                             generate user kubeconfig, stored at CK8S_CONFIG_PATH/user/"
+    echo "  admin-kubeconfig                            generate cluster-admin kubeconfig, stored at CK8S_CONFIG_PATH/.state/"
```

Comment:
Is it necessary to store it in `.state`? Isn't that why we want it in the first place, to not need to save the kubeconfig in the repo.

I guess maybe this is a start, and a separate issue would be to remove the assumption of having the kubeconfig in this location and rely on `KUBECONFIG` instead.

Reply:
I interpreted [this comment from Cristian](https://github.com/elastisys/compliantkubernetes-apps/issues/323#issuecomment-801784144) as "put it in .state for now and fix it later when we move towards actually implementing the ADR fully", so that would be a separate issue like you said.

Reply:
```suggestion
    echo "  admin-kubeconfig <wc|sc>                    generate cluster-admin kubeconfig, stored at CK8S_CONFIG_PATH/.state/"
```